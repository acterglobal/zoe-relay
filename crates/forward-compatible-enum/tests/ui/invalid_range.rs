use forward_compatible_enum::ForwardCompatibleEnum;

#[derive(ForwardCompatibleEnum)]
#[forward_compatible(range = "invalid")]  // Invalid range format!
pub enum InvalidRange {
    #[discriminant(1)]
    Variant1,
    Unknown { discriminant: u32, data: Vec<u8> },
}