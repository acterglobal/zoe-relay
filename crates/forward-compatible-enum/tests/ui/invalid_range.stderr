error: range must be in format 'min..max'
 --> tests/ui/invalid_range.rs:3:10
  |
3 | #[derive(ForwardCompatibleEnum)]
  |          ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `ForwardCompatibleEnum` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/invalid_range.rs:9:2
  |
9 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/invalid_range.rs`
